.github/workflows/bump-openapi.yml

```diff
@@ -0,0 +1,31 @@
+name: bump-openapi
+on:
+  workflow_dispatch:
+    inputs:
+      sentry-api-schema-sha:
+        description: 'The "sentry-api-schema" SHA to run against'
+        required: false
+  push:
+    branches:
+      - master
+
+jobs:
+  bump:
+    name: Bump SHA referencing "sentry-api-schema" to ${{ github.event.inputs.sentry-api-schema-sha }}
+    runs-on: ubuntu-16.04
+    if: ${{ github.event.inputs.sentry-api-schema-sha != '' }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Bump latest SHA
+        sed -i "" 's|^const SENTRY_API_SCHEMA_SHA =.*$|const SENTRY_API_SCHEMA_SHA = "${{ github.event.inputs.sentry-api-schema-sha }}"|g' src/gatsby/utils/resolveOpenAPI.ts
+
+    - name: Git Commit & Push
+      uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        repository: sentry-docs
+        branch: master
+        commit_message: Latest SHA from getsentry/sentry-api-schema
+        commit_user_email: bot@getsentry.com
+        commit_user_name: openapi-getsentry-bot
```

Comment:
1) We should either fork or use a commit hash for external actions (that are not by github), the reason being that `v4` here refers to either a tag or branch in github, both are mutable, so can potentially break our builds.

2) We don't need this action at all, you can use native git commands if you configure the name/email first see https://github.com/getsentry/sentry/blob/cc7b9015a7f43c9e5a909e28c69502c627dd6c00/.github/workflows/release.yml#L49-L50 for an example

3) This won't work if master is a protected branch